localization/languages/cs.json

```diff
@@ -117,7 +117,7 @@
     "phishingErrorVisitAnyway": "Přesto stránky navštívit",
     "phishingErrorLeave": "Opustit tyto stránky",
     /* multiple instances alert */
-    "multipleInstancesErrorMessage": "Vyskytla se chyba. Prosím zavřete všechny ostatní spuštěné instance a restartujte Min.",
```

Comment:
What's the reason for this change?

Reply:
It sounds more civilized to reference Min as a browser. But since I used "browser Min" phrase in other translations, let's use it here as well.